src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -1136,9 +1136,25 @@ Value emitScalarOpFor<ONNXModOp>(ConversionPatternRewriter &rewriter,
 #endif
   }
   if (create.math.isIntegerWithVector(elementType)) {
-    // TODO: implement
-    llvm_unreachable("not support integers at this moment since MLIR integers "
-                     "are signless.");
+    // int diviend, divisor;
+    // rem = diviend % divisor
+    // if (rem == 0)
+    //   return rem
+    // if (diviend < 0)
+    //   rem = rem - divisor  // make rem minus
+    // if (divisor < 0)
+    //   rem = rem + divisor; // make rem minus
+    // return rem;
```

Comment:
The code is little tricky. To understand it, I list out the case and result according to your code (NOT the comment):

1. dividend > 0 and divisor >0, rem unchanged
2. dividend < 0 and divisor <0, rem = (rem+divisor) + divisor = rem + 2*divisor
3. dividend < 0 and divisor > 0, rem = rem + divisor = rem + |divisor|
4. dividend > 0 and divisor < 0, rem = rem+divisor = rem - |divisor|

Is the second case correct? Fix the difference of the comment and code first, please.
Another question: do we need to check eq to zero again at line 1156?